plugins/janus_audiobridge.c

```diff
@@ -1919,8 +1924,25 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->ssrc = 0;
 		pkt->timestamp = ntohl(rtp->timestamp);
 		pkt->seq_number = ntohs(rtp->seq_number);
+		/* Any audio level extension to quickly check if this is silence? */
+		pkt->silence = FALSE;
+		if(participant->extmap_id > 0) {
+			int level = 0;
+			if(janus_rtp_header_extension_parse_audio_level(buf, len, participant->extmap_id, &level) == 0) {
+				pkt->silence = (level == 127);
```

Comment:
As a side note, this can also be dynamic, if as you suggested one introduces history in this. Anyway, that's way more complex that I can handle in an effective way, I believe, so if you have more experience on these aspects a PR after we merge this would be very welcome!